packages/zero-client/src/client/options.ts

```diff
@@ -28,15 +28,22 @@ export interface ZeroOptions<
   server?: string | null | undefined;
 
   /**
-   * A string token to identify and authenticate the user, a function that
-   * returns such a token, or undefined if there is no logged in user.
-   *
-   * If the zero-cache determines the token is invalid (expired, can't be
-   * decoded, bad signature, etc):
-   * 1. if a function was provided Zero will call the function to get a new
-   *    token with the error argument set to `'invalid-token'`.
-   * 2. if a string token was provided Zero will continue to retry with the
-   *    provided token.
+   * A JWT to identify and authenticate the user. Can be provided as either:
+   * - A string containing the JWT token
+   * - A function that returns a JWT token
+   * - `undefined` if there is no logged in user
+   *
+   * Token validation behavior:
+   * 1. **For function providers:**  
+   *    When Zero-cache reports that a token is invalid (expired, malformed,
```

Comment:
Use "`zero-cache`" (with the backticks and lower-case) when referring to this component.